src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -106,17 +114,17 @@ def generate_MCP_1_default_config(self, filename):
                                                                 pal_low_frequency,
                                                                 pal_high_frequency,
                                                                 'test_user_1')
-    # Define DPA's 
+    # Define DPAs
     dpa_1 = {
-       'dpaId': 'dpa_east_4',
+       'dpaId': 'east_dpa4',
```

Comment:
Please use the correct DPA names.

Reply:
Updated   DPA ids as "east_dpa_4", "east_dpa_5" and   "east_dpa_6"